addons/account/static/src/js/reconciliation/reconciliation_renderer.js

```diff
@@ -525,7 +525,7 @@ var LineRenderer = Widget.extend(FieldManagerMixin, {
         }
         return this.model.makeRecord('account.bank.statement.line', [field], {
             partner_id: {
-                domain: ["|", ["is_company", "=", true], ["parent_id", "=", false], "|", ["customer", "=", true], ["supplier", "=", true]],
```

Comment:
Now we are gonna see every `res.partner`, even if they have never been part of your accounting?